tardis/plasma/properties/level_population.py

```diff
@@ -32,64 +33,65 @@ def calculate(level_population_fraction, ion_number_density):
             level_population_fraction.index.droplevel(2)].values
         return level_population_fraction * ion_number_density_broadcast
 
-class LevelPopulationNLTE(ProcessingPlasmaProperty):
+class LevelBoltzmannFactorNLTE(ProcessingPlasmaProperty):
+    name = 'level_boltzmann_factor_nlte'
+
     @staticmethod
-    def calculate(self):
+    def calculate(nlte_input, beta_sobolev, j_blues, levels, nlte_data,
```

Comment:
Ok - I've been using it as a property. What would you suggest instead? Separate properties for the coronal approximation, classical nebular approximation, etc.?